cpp/src/arrow/util/compression.h

```diff
@@ -27,7 +27,7 @@
 namespace arrow {
 
 struct Compression {
-  enum type { UNCOMPRESSED, SNAPPY, GZIP, LZO, BROTLI, ZSTD, LZ4 };
+  enum type { UNCOMPRESSED, SNAPPY, GZIP, BROTLI, ZSTD, LZ4, LZO };
```

Comment:
We should probably remove LZO, but I think this is exposed in parquet-cpp right now

Reply:
It is part of the Parquet standard, so Parquet will need it (altough we don't support it at the moment). I'm not aware how widly it is used, yet no user complained.